intranet/apps/dataimport/management/commands/year_cleanup.py

```diff
@@ -49,9 +52,7 @@ def handle(self, *args, **options):
         turnover_date = datetime.datetime(current_year, 7, 1)
         self.stdout.write("Turnover date set to: {}".format(turnover_date.strftime("%c")))
 
-        if not self.chk(
-            "SENIOR_GRADUATION_YEAR = {} in settings/__init__.py".format(new_senior_year), settings.SENIOR_GRADUATION_YEAR == new_senior_year
-        ):
+        if not self.chk("senior_grad_year = {}".format(new_senior_year), options["senior_grad_year"] == new_senior_year):
```

Comment:
I think it would be better to set `options["senior_grad_year"]` to a variable and use this variable elsewhere in this function.